### ISSUE TITLE
Add clause for 'Specifications using RDFS, SHACL, and OWL to state requirements'

### ISSUE THREAD

**Issue by dr-shorthair** [opengeospatial/ogc-modspec#18]
Alongside the clauses for UML, XML Schema etc.

**Comment by dr-shorthair** [comment]
Maybe also add SHACL here.

**Comment by cnreediii** [comment]
@dr-shorthair That should be "standards using . . . :-) That said, while I do not disagree with the suggestion, who will write these clauses? And what are the implications with using Metanorma? Again, another topic for discussion!

**Comment by cmheazel** [comment]
Consider re-structuring the ModSpec as a multi-part standard.  A core (platform independent) Part 1 and technology specific parts 2-n.  This would mirror how we write other standards.

**Comment by cnreediii** [comment]
@cmheazel Totally agree! This issue (18) is actually two sub-issues: ModSpec version 2 should be in Parts as you suggest and some future parts could/should be JSON, OWL, etc. 

Perhaps Issue 18 can be resolved in the near term.

**Comment by rob-metalinkage** [comment]
I don't think we make this mandatory, but specifications that can use a technology appropriate machine readable and testable constraints are going to be far better than those relying on discovering and parsing and interpreting text alone.

**Comment by cmheazel** [comment]
RDFS, SHACL, and OWL are usually used together.  Suggest that we document these requirements as three sections of one part.

**Comment by cnreediii** [comment]
Currently added as note in Future Work clause.
